content/_data/changelogs/weekly.yml

```
@@ -22504,6 +22504,28 @@
         pr_title: allow to change the icon size of the node overview table
         message: |-
           Allow icon size to be changed in the node overview table.
+      - type: bug
+        category: bug
+        pull: 8836
+        authors:
+          - basil
+        pr_title: Use ConcurrentHashMap#computeIfAbsent in JarCacheSupport
+        references:
+          - url: https://github.com/jenkinsci/remoting/pull/713
+            title: Pull request 713
+        message: |-
+          Remove code that may have caused an agent-side hang under a rare race condition.
+      - type: bug
+        category: bug
+        pull: 8847
+        authors:
+          - basil
+        pr_title: Avoid continual thread replacement in JarCacheSupport
+        references:
+          - url: https://github.com/jenkinsci/remoting/pull/717
+            title: Pull request 717
```

```suggestion
            title: Remoting PR 717
```
Same as above, aligns with Remoting entries in LTS changelog